kafka-client/src/main/java/dev/responsive/kafka/internal/stores/KeyValueOperations.java

```diff
@@ -23,10 +23,6 @@
 
 public interface KeyValueOperations extends Closeable, RecordBatchingStateRestoreCallback {
 
-  void register(ResponsiveStoreRegistry storeRegistry);
```

Comment:
It seems silly to have the registry still live in the KeyValue/WindowStore while moving the registration to the KeyValue/WindowOperations class and just call through these extra de/register APIs.

Basically I moved all the Responsive logic into the Operations implementations, so that the Window/KeyValueStores themselves are now a fairly clean implementation of only what's relevant to Kafka Streams and the Responsive stuff is all encapsulated inside the various Operations